src/apiClient.js

```diff
@@ -2031,6 +2031,68 @@ class ApiClient {
         });
     }
 
+    uploadItemSubtitle(itemId, language, isForced, file) {
+        if (!itemId) {
+            throw new Error('null itemId');
+        }
+
+        if (!language) {
+            throw new Error('null language');
+        }
+
+        if (typeof isForced !== 'boolean') {
+            throw new Error('Invalid isForced value.');
+        }
+
+        if (!file) {
+            throw new Error('File must be a subtitle file.');
+        }
+
+        const format = file.name.substring(file.name.lastIndexOf('.') + 1).toLowerCase();
+
+        if (!['sub', 'srt', 'vtt', 'ass', 'ssa'].includes(format)) {
```

Comment:
Use `endsWith` ([MDN](https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/npm?view=azure-devops&tabs=yaml))

Reply:
`fomat` is also used as parameter in the ajax request.
Using endsWith will not remove the need to parse the file extension.

Reply:
Fair enough, then this is good to go when the backend part is merged.

I'll likely add JSDOC comments and @since once we know which Jellyfin version this will go in, if that's okay.